api/src/main/java/org/openmrs/module/sync2/client/rest/RestClientFactory.java

```diff
@@ -0,0 +1,16 @@
+package org.openmrs.module.sync2.client.rest;
+
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+
+public class RestClientFactory extends HttpComponentsClientHttpRequestFactory {
+
+    private final static int connectionRequestTimeout = 1000;
+    private final static int connectTimeout = 15 * 1000;
+    private final static int readTimeout = 30 * 1000;
```

Comment:
These 3 constants names should be Upper-case with the underscore between words. Additionally, you can use 1000 as const.